migrate/troubleshooting.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Troubleshooting known issues
+excerpt: Troubleshooting known issues in database migrations
+products: [cloud, self_hosted]
+keywords: [backups, restore]
+tags: [recovery, logical backup, pg_dump, pg_restore]
+---
+
+import OpenSupportRequest from "versionContent/_partials/_migrate_open_support_request.mdx"
+
+# Troubleshooting known issues
+
+## Large migrations blocked
+
+Timescale's platform automatically manages the underlying disk volume. Due to
+platform limitations, it is only possible to resize the disk once every six
+hours. Depending on the rate at which you're able to copy data, you may be
```

Comment:
I wonder if this note should be given at migration/index page. Keeping the note here means a user will look into "Troubleshooting" when he has faced a migration error. At this moment, he has spend time in migration and used some credits. We should inform him first about the possibility of such errors before he starts a migration.

Reply:
Thanks, I've left this here, but also mentioned it on each of the step-by-step pages.

Reply:
Sorry, I am unable to find the caution for "large migrations" in the pages for dual-writes for Timescale/Postgres. Can you please help me point it out? I think this is important for users before they begin the migration journey.